### PR TITLE
feature/my-folder

### PR DESCRIPTION
**Pull request by manuelbarzi**

### CODE REVIEW

File: staff/john-doe/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[github](https://github.com/johndoe)
```
Comment (manuelbarzi) [inline staff/john-doe/README.md]: hey should eat cakes with tea

Comment (RoseKoning) [inline staff/john-doe/README.md]: I love cheesecake

Comment (pujabhattacharya1) [inline staff/john-doe/README.md]: First learn to bake some

Comment (davide-st) [inline staff/john-doe/README.md]: first review

**Review by davide-st — COMMENTED** [review]
test

**Pull request merged**